proto/Makefile

```diff
@@ -0,0 +1,81 @@
+SHELL := /usr/bin/env bash
+
+.PHONY: all
+all: \
+	clang-format \
+	buf-lint \
+	buf-breaking \
+	api-linter-lint \
+	buf-generate \
+	buf-generate-example
+
+include tools/api-linter/rules.mk
+include tools/buf/rules.mk
+include tools/clang-format/rules.mk
+include tools/protoc-gen-go-grpc/rules.mk
+
+build/protoc-gen-go: ../go.mod
+	$(info [$@] building binary...)
+	@go build -o $@ google.golang.org/protobuf/cmd/protoc-gen-go
+
+build/protoc-gen-go-aip: ../go.mod
+	$(info [$@] building binary...)
+	@go build -o $@ go.einride.tech/aip/cmd/protoc-gen-go-aip
```

Comment:
So this is to build what we would normally code generate in the proto repo?

Reply:
this is to build the `protoc-gen-go-aip` code generator plugin - we build it like this to ensure we build it with the same version as the repo depends on